app/controllers/planning_applications/assessment/local_policy_areas_controller.rb

```diff
@@ -0,0 +1,69 @@
+# frozen_string_literal: true
+
+module PlanningApplications
+  module Assessment
+    class LocalPolicyAreasController < AuthenticationController
+      before_action :set_planning_application
+      before_action :set_local_policy, only: %i[show edit update new]
+      before_action :set_local_policy_area, only: %i[edit show]
+
+      def show
+        set_local_policy_area
+      end
+
+      def new
+        @local_policy_area = LocalPolicyArea.new
+      end
+
+      def edit
+        set_local_policy_area
+      end
+
+      def destroy
+        @local_policy_area = LocalPolicyArea.find(params[:id])
+        @local_policy_area.destroy!
+
+        redirect_to new_planning_application_assessment_local_policy_path(@planning_application)
+      end
+
+      def create
+        set_local_policy
+        @local_policy_area = LocalPolicyArea.new(local_policy_params.except(:policy))
+
+        if @local_policy_area.save
+          if @planning_application.local_policy.present?
+            redirect_to edit_planning_application_assessment_local_policy_path(@planning_application, @planning_application.local_policy)
+          else
+            redirect_to new_planning_application_assessment_local_policy_path(@planning_application)
+          end
+        else
+          render :new
+        end
```

Comment:
We should always use `respond_to` blocks in actions so that we define the formats that we accept. If we don't then someone can edit the url to include a format like `.json` and Rails will try to render it.